generator/src/main/java/com/scottlogic/deg/generator/commandline/CommandLineBase.java

```diff
@@ -36,7 +36,7 @@
     @CommandLine.Option(
         names = { "--disable-schema-validation" },
         description = "Disables schema validation")
-    boolean disableSchemaValidation = false;
+    boolean disableSchemaValidation = true;
```

Comment:
Once this change is in place we will have no way of enabling the schema validation?
I guess we also need to reverse the option to `--enable-schems-validation`

Reply:
Agree, i'd suggest the following changes:
- return this to being defaulted to `false`
- rename the field to 'enableSchemaValidation', update the description as appropriate
- remove the negation in `isSchemaValidationEnabled` in both `GenerateCommandLine` and `VisualiseCommandLine`